config/docker-val.config.src

```diff
@@ -18,7 +18,7 @@
    {block_sync_batch_limit, 100},
    {max_inbound_connections, 32},
    {snapshot_memory_limit, 2048},
-   {key, undefined},
+   {key, {tpm, [{key_path, "HS/SRK/MinerKey"}]}},
```

Comment:
Maybe it would make sense here to add an `{M, F, A}` style callback method for key acquisition instead of continuing to add more special cases for all the different ways to load key material. Thinking like `{callback, {Mod, Func, Args}}` would be a nice escape hatch